lib/crypt-pbkdf1-sha1.c

```diff
@@ -165,8 +165,10 @@ crypt_sha1crypt_rn (const char *phrase, size_t phr_size,
       hmac_sha1_process_data (hmac_buf, SHA1_SIZE, pwu, pl, hmac_buf);
     }
   /* Now output... */
-  pl = (size_t)snprintf ((char *)output, out_size, "%s%lu$%.*s$",
-                         magic, iterations, (int)sl, setting);
+  dl = snprintf ((char *)output, out_size, "%s%lu$%.*s$",
```

Comment:
there's another one above (L156)

Reply:
I only looked in the ones that are casted to `size_t`, but you are right, it might make sense to go through all `snprintf` calls and make sure return value is checked for negativity.

Reply:
@fweimer-rh thanks for looking, I'll check.